server/@types/frontendComponentApiClient/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'frontendComponentApiClient' {
+  export interface Component {
+    html: string
+    css: string[]
+    javascript: string[]
+  }
+}
```

Comment:
To follow convention with what we've already done, could you create both an api and apiclient folder please? Something like:

frontendComponentApi/index.d.ts
```
export interface Component {
  html: string
  css: string[]
  javascript: string[]
}
```

and then frontendComponentApiClient.index.d.ts
```
declare module 'frontendComponentApiClient' {
  import Component from '../frontendComponentApi'

  export type FrontendComponent = Component
}
```

I know it might seem like writing code for the sake of it, and it might not be adding much; but it would be consistent with everything else we already have, plus would allow for us to import their swagger spec (if they ever produce one) without too much hassle